ada/ast.py

```diff
@@ -13879,6 +13930,52 @@ def env_elements_baseid():
             )
         ))
 
+    @langkit_property(return_type=Equation)
+    def denotes_the_property_function(subp_decl=T.SubpDecl.entity):
```

Comment:
There is something weird: since the equation returned is either `LogicTrue()` or `LogicFalse()`, it seems to me like you don't even need an equation here: you can simply return a `Bool` and use to filter out subprogram candidates directly in `stable_properties_assoc_equation`.